caropa_app/management/commands/create_fake_data.py

```diff
@@ -13,4 +13,5 @@ def add_arguments(self, parser):
     """
     def handle(self, *args, **options):
         number_of_records = options['number_of_records']
+        call_command('create_categories')
```

Comment:
Don't exists the '`create_categories`' command, if you try to run `python manage.py create_fake_data` don't works

Reply:
Done!